portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['You can tell me Brandocean', 'I like watching anime', 'I have 3 dogs', 'I study at Tec de Monterrey in Cuernavaca, Morelos'];
```

Comment:
[4.4](https://google.github.io/styleguide/jsguide.html#formatting-column-limit): JavaScript code has a column limit of 80 characters.

I suggest placing each value on a separate line like in [this example](https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma).